code/ui/blocks/src/blocks/internal/InternalCanvas.stories.tsx

```diff
@@ -109,7 +109,7 @@ export const LayoutAsChildProp: Story = {
   render: (args) => {
     return (
       <Canvas {...args}>
-        <StoryComponent of={ButtonStories.Primary} parameters={{ layout: 'fullscreen' }} />
+        <StoryComponent of={ButtonStories.Primary} />
```

Comment:
I thought `parameters` was not a thing anymore, given the documentation. It seems like they still have effect, in fact [Chromatic detected regressions](https://www.chromatic.com/test?appId=635781f3500dd2c49e189caf&id=65971f79f5f5b426b598524b) based on this removal. Can you assist @JReinhold ?

Reply:
Well, it's complicated... It _only_ works for the `layout` parameter, and it's in fact a feature of `Canvas` and not of `Story` - [`Canvas` reads the prop from its children](https://github.com/storybookjs/storybook/blob/yann/remove-deprecated-story-block-props/code/ui/blocks/src/blocks/Canvas.tsx#L131-L137).

Hopefully that deprecated functionality of `Canvas` will also be removed soon, but there is some other work to be done before that. So worst case is that that functionality will be kept around in v8 because we don't get to it in time.

I see two paths forward here:

1. Remove the code in `Canvas` that I linked to above, removing the functionality of reading from children but keep the other stuff around. Then when/if we get around to cleaning up `Canvas` too, we'll remove all of the deprecated code.
2. Keep the functionality in tact until we remove code from `Canvas` too. That means adding a deprecated prop type to `Story` a la `parameters?: { layout: Layout }`, to ensure the types won't break for users. Then add a comment in `Canvas` reminding us to remove that type.

I'm fine with either solutions.

Reply:
I chose solution 1.